contrib/librdkafka-cmake/config.h.in

```diff
@@ -66,7 +66,7 @@
 #cmakedefine WITH_SASL_OAUTHBEARER 1
 #cmakedefine WITH_SASL_CYRUS 1
 // crc32chw
-#if !defined(__PPC__) && !defined(__riscv) && !defined(__aarch64__)
+#if !defined(__PPC__) && !defined(__riscv) && !defined(__aarch64__) && !defined(__s390x__)
 #define WITH_CRC32C_HW 1
```

Comment:
Does s390x CPU have CRC32-C instruction?
I remember you told me the opposite: https://github.com/ClickHouse/ClickHouse/pull/40948

Reply:
@alexey-milovidov I am checking with our compiler team on this